platform-sdk/sdk/settings.txt

```diff
@@ -44,5 +44,4 @@ prometheus.endpointEnabled,               false
 
 loadKeysFromPfxFiles,                     false
 crypto.enableNewKeyStoreModel,            false
-crypto.cpuVerifierThreadRatio,            0.1

```

Comment:
did you intend to include this change in this PR?

Reply:
Yes. It is not currently used. I removed it from the config record in another PR, but forgot to update the `settings.txt` we use for local testing.